server/selinux/server/pulp-celery.te

```diff
@@ -91,7 +106,6 @@ logging_send_syslog_msg(celery_t)
 miscfiles_read_localization(celery_t)
 miscfiles_manage_cert_dirs(celery_t)
 sysnet_read_config(celery_t)
-
```

Comment:
I think this whitespace was useful. It also is consistent with the rest of the policy to have a line of whitespace after a policy block.